src/components/Header/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  title: string
+  setTitle: (newTitle: string) => void
+  onSaveNewTodo: () => void,
+  tempTodo: Todo | null,
+};
+
+export const Header: React.FC<Props> = ({
+  title,
+  setTitle,
+  onSaveNewTodo,
+  tempTodo,
+}) => {
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    onSaveNewTodo();
+  };
+
+  return (
+    <header className="todoapp__header">
+      {/* this buttons is active only if there are some active todos */}
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button type="button" className="todoapp__toggle-all active" />
```

Comment:
The button should be conditionally rendered based on the presence of active todos. Right now it is always visible with a comment that suggests it should only be visible under certain conditions.

Reply:
As far as I understood, this task requires just add and remove actions. Toggle all button is a part of the next task.